.github/workflows/check.yaml

```diff
@@ -108,7 +108,15 @@ jobs:
           juju version
           juju controllers | grep Version -A 1 | awk '{print $9}'
 
+      - name: Build the charm
+        run: charmcraft -v pack
+
       - name: Build charm and run tests
-        run: make integration
+        run: |
+          export CHARM_PATH_JAMMY="$(pwd)/$(ls | grep '.*22.04.*\.charm$')"
+          echo "$CHARM_PATH_JAMMY"
+          export CHARM_PATH_FOCAL="$(pwd)/$(ls | grep '.*20.04.*\.charm$')"
+          echo "$CHARM_PATH_FOCAL"
```

Comment:
I'm not understanding when this is going to be used.

The charmcraft now is:

```yaml
bases:
  - build-on:
    - name: ubuntu
      channel: "22.04"
    run-on:
    - name: ubuntu
      channel: "22.04"
```

And we are not using ubuntu 20.04 in the CI

Reply:
Ah yep sorry for the confusion - please see https://github.com/canonical/charm-userdir-ldap/pull/47#discussion_r1764142777 for the explanation.  I can add a comment to https://github.com/canonical/solutions-engineering-automation/pull/84 too.